model/db-common.yaml

```diff
@@ -1,22 +1,39 @@
 groups:
-  - id: attributes.db.client
+  - id: attributes.db.client.minimal
     type: attribute_group
     brief: 'Database Client attributes'
     attributes:
-      - ref: db.namespace
-        requirement_level:
-          conditionally_required: If available.
-      - ref: db.collection.name
```

Comment:
can `db.collection.name` be part of the minimal set?

Reply:
It does not apply to redis or memcached